bundles/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/models/v2/dto/request/HeatSetPoint.java

```diff
@@ -43,12 +43,15 @@ public class HeatSetPoint {
         timeUntil = null;
     }
 
+    @SuppressWarnings("unused")
```

Comment:
Why is this suppression needed?

Reply:
As this is a DTO class for sending a request this might be part of the json model to send to the server. I did not want to touch it, but i do want to get rid of the compiler warning. Nothing more. It would be nice if SAT would detect such DTO class and hides the unsued warning when the property has been annotated with SerializedName.